public/tradingview/colors.css

```diff
@@ -20,9 +20,9 @@
 
   --color-accent: #6966FF; /* ColorToken.Purple1; */
 }
-
-/* Theme: Light */
-:root.theme-light {
+  
+  /* Theme: Light */
+  :root.theme-light {
```

Comment:
ok idk why but my vscode also autoformats this and I feel like it shouldn't

Reply:
will remove from this pr t_t